src/Handler.php

```diff
@@ -104,18 +105,45 @@ private function processOperations(array $operations, array $versions, bool $red
 
         foreach ($operations as $offset => $operation) {
             $sql = $this->getBuilder()->build($operation);
+
+            // Determine if transaction is required.
+            $transactionRequired = $operations instanceof ExecOperation;
+
+            // Begin trx if needed.
+            if ($transactionRequired) {
+                var_dump('YO!');
```

Comment:
Try not to mix terminology in comments. Single line comments don't need a period (check previous comment in same block too).

There's a remnant debugging log here too.

Reply:
Oops! Thanks, Handled!